engfrosh_bot/__main__.py

```diff
@@ -148,6 +148,29 @@ async def discord_queue_callback(command: dict):
         logger.warning("Object type %s not supported. Ignoring command" % command["object"])
         return
 
+# Add Bot Listner for when someone gets a role
+intents = discord.Intents.default()
+intents.members = True
+intents.presences = True
+client = discord.Client(intents=intents)
```

Comment:
Don't create a new `client`, it already exists, and it's fine to edit it if needed.